### PR TITLE
OP-1136 | Add diagnosis validation upon Admission and Discharge

### PR DESCRIPTION
**Pull request by ArnaudFonzam**
See OP-1136.

### CODE REVIEW

File: bundle/language_en.properties [hunk 1]
```diff
@@ -42,8 +42,12 @@ angal.admission.deliverytype.border
 angal.admission.diagnosisin.col                                                                        = Diagnosis In
 angal.admission.diagnosisincannotbeempty.msg                                                           = Diagnosis -IN- cannot be empty.
 angal.admission.diagnosisinstar.border                                                                 = Diagnosis -IN- *
+angal.admission.diagnosisinisnotallowed.msg                                                            = Diagnosis -IN is not allowed
```
Comment (dbmalkovsky) [inline bundle/language_en.properties:45]: Messages like this should end in a period like like 43 above.

File: bundle/language_en.properties [hunk 2]
```diff
@@ -42,8 +42,12 @@ angal.admission.deliverytype.border
 angal.admission.diagnosisin.col                                                                        = Diagnosis In
 angal.admission.diagnosisincannotbeempty.msg                                                           = Diagnosis -IN- cannot be empty.
 angal.admission.diagnosisinstar.border                                                                 = Diagnosis -IN- *
+angal.admission.diagnosisinisnotallowed.msg                                                            = Diagnosis -IN is not allowed.
```
Comment (mwithi) [inline bundle/language_en.properties:45]:
```suggestion
angal.admission.diagnosisinisnotallowed.msg                                                            = Diagnosis -IN- provided is not allowed.
```

File: bundle/language_en.properties [hunk 3]
```diff
@@ -42,8 +42,12 @@ angal.admission.deliverytype.border
...
 angal.admission.diagnosisout.border                                                                    = Diagnosis -OUT-
 angal.admission.diagnosisout.col                                                                       = Diagnosis Out
+angal.admission.diagnosisout1isnotallowed.msg                                                          = Diagnosis Out 1 is not allowed.
+angal.admission.diagnosisout2isnotallowed.msg                                                          = Diagnosis Out 2 is not allowed.
+angal.admission.diagnosisout3isnotallowed.msg                                                          = Diagnosis Out 3 is not allowed.
```
Comment (mwithi) [inline bundle/language_en.properties:50]:
```suggestion
angal.admission.diagnosisout1isnotallowed.msg                                                          = Diagnosis Out 1 provided is not allowed.
angal.admission.diagnosisout2isnotallowed.msg                                                          = Diagnosis Out 2 provided is not allowed.
angal.admission.diagnosisout3isnotallowed.msg                                                          = Diagnosis Out 3 provided is not allowed.
```

**Comment by mwithi** [comment]
Superseded by https://github.com/informatici/openhospital-gui/pull/1938.

**Pull request closed without merging**